src/muse/data/example/trade/settings.toml

```diff
@@ -50,11 +50,13 @@ agents = '{path}/technodata/residential/Agents.csv'
 existing_capacity = '{path}/technodata/residential/ExistingCapacity.csv'
 lpsolver = "scipy"
 constraints = [
+    # Optional, defaults to the constraints below
```

Comment:
Something you sometimes see in example config files is for the default options to be present, but commented out (i.e. in this case `constraints`). Something to think about?